### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by diversityquota**
This is a pull request test for omebe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 README
+update
```
Comment (diversityquota) [inline README.md:2]: But what kind of update?

Comment (diversityquota) [reply README.md:2]: @bitadj

**Review by bitadj — COMMENTED** [review]
Requesting a change

**Review by bitadj — CHANGES_REQUESTED** [review]
requesting changes to PR from diversityquota

**Pull request closed without merging**